README.md

```diff
@@ -1,98 +1,146 @@
+If want to play with our code, these documents should prove useful:
 
-## Branching Strategy
+-   [Running Puppet Server from
+    source](./documentation/dev_running_from_source.markdown)
+-   [Debugging](./documentation/dev_debugging.markdown)
+-   [Puppet Server subcommands](./documentation/subcommands.markdown)
 
-The Branching Strategy for Puppet Server is documented on the wiki [here](https://github.com/puppetlabs/puppet-server/wiki/Branching-Strategy).
+Puppet Server also uses the
+[Trapperkeeper](https://github.com/puppetlabs/trapperkeeper) Clojure framework.
 
-## Issue Tracker
+## Branching strategy
 
-Feature requests?  Found a bug?  Want to see what issues are currently in flight?  Please visit our [Jira project](https://tickets.puppetlabs.com/browse/SERVER).
+Puppet Server's branching strategy is documented on the [GitHub repo
+wiki](https://github.com/puppetlabs/puppetserver/wiki/Branching-Strategy).
+
+## Issue tracker
+
+Have feature requests, found a bug, or want to see what issues are in flight?
+Visit our [JIRA project](https://tickets.puppetlabs.com/browse/SERVER).
 
 ## License
 
-Copyright © 2013 -- 2014 Puppet Labs
+Copyright © 2013---2016 Puppet
 
-Distributed under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html)
+Distributed under the [Apache License, Version
+2.0](http://www.apache.org/licenses/LICENSE-2.0.html).
 
-### Special thanks to
+## Special thanks to
 
-#### Cursive Clojure
+### Cursive Clojure
 
-[Cursive](https://cursiveclojure.com/) is a Clojure IDE based on
-[IntelliJ IDEA](http://www.jetbrains.com/idea/download/index.html).  Several of
-us at Puppet Labs use it regularly and couldn't live without it.  It's got
-some really great editing, refactoring, and debugging features, and the author,
-Colin Fleming, has been amazingly helpful and responsive when we have feedback.
-If you're a Clojure developer you should definitely check it out!
+[Cursive](https://cursiveclojure.com/) is a Clojure IDE based on [IntelliJ
+IDEA](http://www.jetbrains.com/idea/download/index.html). Several of us at
+Puppet use it regularly and couldn't live without it. It's got some really great
+editing, refactoring, and debugging features, and the author, Colin Fleming, has
+been amazingly helpful and responsive when we have feedback. If you're a Clojure
+developer, you should definitely check it out!
 
-#### JRuby
+### JRuby
 
 [JRuby](http://jruby.org/) is an implementation of the Ruby programming language
-that runs on the JVM.  It's a fantastic project, and it is the bridge that allows
-us to run all of the existing Puppet Ruby code while taking advantage of all of
-the advanced features and libraries that are available on the JVM.  We're very
-grateful to the developers for building such a great product and for helping us
-work through a few bugs that we've discovered along the way.
-
-[leiningen]: https://github.com/technomancy/leiningen
```

Comment:
Oh! huh. I was reading it as a thing we were thanking, but yeah, it's not formatted right for that...